docs/rfc/rfc-19-random-beacon-v2.adoc

```diff
@@ -0,0 +1,157 @@
+:toc: macro
+
+= RFC 19: Random Beacon v2
+
+:icons: font
+:numbered:
+toc::[]
+
+== Background
+
+The last release of the random beacon got its genesis Sep 16th 2020 and worked
+until November 11th when it was stopped with a panic button as a result of an
+unannounced https://github.com/keep-network/keep-core/blob/main/docs/status-reports/2020-11-11-retro-geth-hardfork.adoc[Ethereum
+hard fork].
+
+For the time it was working, the beacon produced more than 1700 relay entries
+and 70 groups.
+
+Both on-chain and off-chain components proved to be successful but - just like
+with any other product - some pain points were identified by the users. This RFC
+aims at describing the pain points and proposing changes that should make the
+beacon easier to operate and more consistent with tBTC v2.
```

Comment:
Can you include a high level summary of the pain points in the background? like maybe bullet points or something

Reply:
Please see https://github.com/keep-network/keep-core/pull/2585/commits/827175f76620b810c5520e3eef2a7af5f9941c81. The two sections below are the major pain points, I should be more explicit about it.